class/subscription.php

```diff
@@ -1401,4 +1431,57 @@ public function cancel_non_recurring_subscription() {
             }
         }
     }
+
+    /**
+     * Migrate subscription sort order data (runs once)
+     * 
+     * This method ensures that all existing subscription packs have a default sort order
+     * set if they don't already have one. It runs only once per site using a version check.
+     *
+     * @since WPUF_SINCE
+     *
+     * @return void
+     */
+    public function migrate_subscription_sort_order() {
```

Comment:
we have a `Upgrades` class in `includes/Admin/Upgrades.php` for this migration task